proto/prysm/v1alpha1/validator.proto

```diff
@@ -662,31 +662,21 @@ message ValidatorCapella {
 
 // ValidatorParticipation stores participation metrics during a given epoch.
 message ValidatorParticipation {
-    // Percentage of validator participation in the given epoch. This field
-    // contains a value between 0 and 1.
-    float global_participation_rate = 1  [deprecated = true];
-
-    // The total amount of ether, in gwei, that has been used in voting.
-    uint64 voted_ether = 2 [deprecated = true];
-
-    // The total amount of ether, in gwei, that is eligible for voting.
-    uint64 eligible_ether = 3 [deprecated = true];
-
     // Total staked gwei that was active (i.e. eligible to vote) during the current epoch.
-    uint64 current_epoch_active_gwei = 4;
+    uint64 current_epoch_active_gwei = 1;
```

Comment:
Do not change field numbers!! 

You should simply delete the fields and then put the field numbers in a reserved section.

It would look something like this and you can put it at the start of the message.
```protobuf
reserved 2, 15, 9 to 11; // Deprecated and removed fields.
```